server.js

```diff
@@ -34,6 +34,7 @@ function renderPage(appHtml,reactHelmet) {
         <link rel="stylesheet" type="text/css" href="/css/mofo-bootstrap.css">
         <link rel="stylesheet" type="text/css" href="/css/font-awesome.min.css">
         <link rel="stylesheet" type="text/css" href="/css/main.css">
+        <link rel="alternate"  type="application/rss+xml" href="https://api.mozillapulse.org/rss/latest" />
```

Comment:
You'll need to do something like this: `<link rel="alternate"  type="application/rss+xml" href="${env.PULSE_API('api/pulse', '')}rss/latest" />` 

The replace is necessary because that "api/pulse" path is appended to the environment variable